### PR TITLE
Simple start

### PR DESCRIPTION
**Pull request by mstumpf585**
Couldn't push because I wasn't a contributor. The code also would not compile with the Wiegand library in the fossball directory so I cloned it to Ardunio/libraries and it ran fine.

### CODE REVIEW

**Comment by mciantyre** [comment]
Great job! I made a couple of quick comments. Absolutely no rush to respond.

File: foosball-arduino.ino [hunk 1]
```diff
@@ -1,19 +1,32 @@
...
 int ins[6];   // Cannot exceed 6 analog inputs 
 bool triggered = false;
 static constexpr int THRESHOLD = 22;
 int count = 0;
 
+
+constexpr char YellowGoalSignal = 'A';
+constexpr char BlackGoalSignal  = 'B';
+constexpr char YellowDropSignal = 'C';
+constexpr char BlackDropSignal  = 'D';
```
Comment (mciantyre) [inline foosball-arduino.ino]: These `constexpr`s may duplicate those in `messages.h` (included above). Can we remove the `messages.h` header altogether?

File: foosball-arduino.ino [hunk 2]
```diff
@@ -26,20 +39,48 @@ void loop()
...
+          break; 
+
+      default:
+        break;
+        
+      }
     }
-    else if (ins[j] >= THRESHOLD && triggered)
+    
+    if((wg.available()))
```
Comment (mciantyre) [inline foosball-arduino.ino]: The check for availability is occurring inside the `for` loop. Is this intentional?

File: foosball-arduino.ino (hunk 1 above)
Comment (pro585code) [inline foosball-arduino.ino]: Yes unless you want to keep messages and do something with it later?

File: foosball-arduino.ino (hunk 2 above)
Comment (pro585code) [inline foosball-arduino.ino]: Oh boy no that's an error will fix

**Pull request closed without merging**